### ISSUE TITLE
Editor 

### ISSUE THREAD

**Issue by ddutchie** [Unity-Technologies/AutoLOD#33]
Hello Unity Labs. 

I'm having an issue using this. I have cloned, pulled dependencies and imported into a project. 

But whenever It tries to run it logs this: 

```
Can't add script behaviour LODVolume because it is an editor script. To attach a script it needs to be outside the 'Editor' folder.
UnityEngine.GameObject:.ctor(String, Type[])
LODVolume:Create() (at Assets/AutoLOD/Scripts/LODVolume.cs:187)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()
```

and this:

```
NullReferenceException: Object reference not set to an instance of an object
UnityEditor.Experimental.AutoLOD.SceneLOD+<UpdateOctree>c__Iterator2.MoveNext () (at Assets/AutoLOD/Scripts/Editor/SceneLOD.cs:338)
UnityEngine.Experimental.AutoLOD.TimedEnumerator.MoveNext () (at Assets/AutoLOD/Scripts/Helpers/TimedEnumerator.cs:38)
UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress) (at C:/buildslave/unity/build/Runtime/Export/Coroutines.cs:17)
```

Any ideas?

**Comment by fpywzywzy** [comment]
the same with you

**Comment by amirebrahimi** [comment]
What version of Unity?

**Comment by fpywzywzy** [comment]
unity 2017.4.6f1

**Comment by Alverik** [comment]
Same on Unity 2018.2.02f, LOD meshes are generated properly but I get an error when trying to "Activate SceneLOD".

**Comment by Alverik** [comment]
OK, it's something to do with the _AutoLOD.dll, when I tick Windows x64 (my OS) in the Include platforms section and restart Unity, then SceneLOD starts working, but then it's not possible to Build the project. I notice the LODVolume script and other components are freely using some editor only API (like a few attributes and the EditorUtility class, etc)............. was it ever even possible to build a project using the SceneLOD feature?

**Comment by amirebrahimi** [comment]
Confirmed the problem was introduced after changing assembly definition files. Now the assembly is added back to builds. Fixed with 1f5acb9 

SceneLOD is untested in builds, but theoretically should work as long as objects don't move (LOD generation doesn't work in the runtime, currently).
